tests/macros/src/test/scala/scala/macros/tests/scaladays/SerializeSuite.scala

```diff
@@ -5,14 +5,12 @@ import org.junit._
 import org.junit.runner._
 import org.junit.runners._
 import org.junit.Assert._
-import scala.macros.config._
 
 @RunWith(classOf[JUnit4])
 class SerializeSuite {
   @Test
   def simple: Unit = {
     case class C(x: Int, y: String)
-    def serialize[T: Serialize](x: T): String = implicitly[Serialize[T]].apply(x)
-    assertEquals("""{ "x": 40, "y": "2" }""", serialize(C(40, "2")))
+    assertEquals("""{ "x": 40, "y": "2" }""", Serialize.materialize[C](C(40, "2")))
```

Comment:
Nice, I like semantic tests.

Reply:
The `syntax(a.b) == "a.b"` tests are strictly speaking just as much semantic tests 😜 

I agree however that we should focus on "real-world" macros instead of synthetic cases. This PR was really only for me to understand things better.